Bio/PDB/FragmentMapper.py

```diff
@@ -24,9 +24,19 @@
 The following example uses the library with 10 fragments of length 5.
 The library files can be found in directory 'fragment_data'.
 
+    >>> from Bio.PDB.PDBParser import PDBParser
+    >>> from Bio.PDB.FragmentMapper import FragmentMapper
+    >>> parser = PDBParser()
+    >>> structure = parser.get_structure("1a8o", "PDB/1A8O.pdb")
     >>> model = structure[0]
-    >>> fm = FragmentMapper(model, lsize=10, flength=5, dir="fragment_data")
-    >>> fragment = fm[residue]
+    >>> fm = FragmentMapper(model, lsize=10, flength=5, fdir="PDB/fragment_data")
```

Comment:
Where is this directory? Does it get created?

I note that https://github.com/biopython/biopython/blob/master/Tests/test_PDB_FragmentMapper.py doesn't cover this argument (it probably should do).

Reply:
Ah, I see you are adding ``Tests/Bio/fragment_data/lib_10_z_5.txt`` to the source code.